### PR TITLE
Improves the service descriptor file

### PR DESCRIPTION
**Pull request by rodrigorigotti**
Improves the service descriptor file so a faulty call is corrected and several common structutes are moved to a separate model.

### CODE REVIEW

**Review by tzengerink — APPROVED** [review]

**Comment by lcobucci** [comment]
@rodrigorigotti I honestly have no context here so I'm unable to give any review

**Comment by rodrigorigotti** [comment]
@lcobucci this is the PHP SDK for our Public API. There's a descriptor file which states what are the endpoints of the API and the parameters for each one of them.

There was a bug in the `WebsiteCampaignResultItems` model so the customers couldn't use the `GetWebsiteCampaignResults` command. I made the fix and also removed some of the redundancies in the description file so it's smaller and easier to understand.

**Review by rdohms — CHANGES_REQUESTED** [review]
Please split the commits, its not clear what the bug was and what the reformatting is.

File: src/Usabilla/API/Description/UsabillaDescription.php [hunk 1]
```diff
@@ -226,139 +175,7 @@ public function __construct()
                         'lastTimestamp' => [ '$ref' => 'LastTimestampAttribute' ],
                     ],
```
Comment (rdohms) [inline src/Usabilla/API/Description/UsabillaDescription.php:176]: does this not lose the item definition?

**Review by rdohms — CHANGES_REQUESTED** [review]

File: src/Usabilla/API/Description/UsabillaDescription.php (hunk 1 above)
Comment (rodrigorigotti) [reply src/Usabilla/API/Description/UsabillaDescription.php:176]: This model already existed, it wasn't just being used here.

Comment (rdohms) [reply src/Usabilla/API/Description/UsabillaDescription.php:176]: yes but the definitions you deleted had a `'items'` key you that is now gone?

Comment (rdohms) [reply src/Usabilla/API/Description/UsabillaDescription.php:176]: nevermind, github was confusing me.

**Review by rdohms — APPROVED** [review]

**Pull request merged**